packages/core/src/components/dropdown/test/error/dropdown.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/dropdown/test/error/index.html';
+
+test.describe('tds-dropdown-error', () => {
+  test('renders error dropdown correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const dropdown = page.getByTestId('tds-dropdown-testid');
+    await expect(dropdown).toHaveCount(1);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('find helper text and check if visible and have icon', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const helperText = page.getByText(/Helper text/);
+    await expect(helperText).toBeVisible();
+    const helperTextIcon = helperText.locator('tds-icon');
```

Comment:
Hmm this is actually testing an implementation detail, since the tds-icon is added inside of our component declaration. 
I think we could leave this out and rely on the screenshot, WDYT?

Reply:
It is a bit of an edge case though, as we're unlikely to use any other element for the icon in the future 🤔

Reply:
No you are right, I do getByRole('img') instead so that we can use any icon there instead